kits/peer-demo-showcase/prompts/showcase-submission-flow_instructor-llmnode-297_system_0.md

```
@@ -0,0 +1,7 @@
+You are analyzing a GitHub project README. Extract the following information from this content:
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)